framework/client/code-styleguide.rst

```diff
@@ -0,0 +1,110 @@
+Coding Styleguide
+=================
+
+General
+-------
+
+**ESlint** Most code styles are enforced by eslint. The Travis CI build
+will fail if not every ESlint rule is fulfilled. Active ESlint rules can
+be seen here:
+https://github.com/tocco/tocco-client/blob/master/.eslintrc
+
+Folders and structure
+---------------------
+
+Naming conventions folders
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+========== ====== ==============
+What       Case   Example
+========== ====== ==============
+Packages   Kebab  entity-browser
+Components Pascal SearchField
+Module     Camel  searchForm
+Any other  Kebab  test-data
+========== ====== ==============
+
+Actions
+-------
+
+-  Wrap arguments in payload attribute
+-  Use arrow functions
+-  Returning object literals (no return statement used)
+
+.. code:: js
```

Comment:
Should be ` .. code-block:: js`  instead of ~~` .. code:: js `~~.

`block` always uses default formatting (the `js` parameter has no effect). You have to use `code-block` for `js` to be honored.